File: airbyte-integrations/connectors/source-postgres/src/test-integration/java/io/airbyte/integrations/io/airbyte/integration_tests/sources/PostgresSourceAcceptanceLegacyCtidTest.java
/*
 * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
 */

package io.airbyte.integrations.io.airbyte.integration_tests.sources;

import io.airbyte.integrations.source.postgres.PostgresTestDatabase.BaseImage;

public class PostgresSourceAcceptanceLegacyCtidTest extends PostgresSourceAcceptanceTest {

  @Override
  protected BaseImage getServerImage() {
    return BaseImage.POSTGRES_12;
  }

}

File: airbyte-integrations/connectors/source-postgres/src/test/kotlin/io/airbyte/integrations/source/postgres/legacy/PostgresSourceAcceptanceLegacyCtidTest.kt
/*
 * Copyright (c) 2026 Airbyte, Inc., all rights reserved.
 */

package io.airbyte.integrations.source.postgres.legacy

import io.airbyte.integrations.source.postgres.legacy.testFixtures.PostgresTestDatabase

class PostgresSourceAcceptanceLegacyCtidTest : PostgresSourceAcceptanceTest() {
    override val serverImage: PostgresTestDatabase.BaseImage
        get() = PostgresTestDatabase.BaseImage.POSTGRES_12
}
